code/model/modelread.cpp

```diff
@@ -53,7 +53,7 @@ flag_def_list model_render_flags[] =
   	 
 int model_render_flags_size = sizeof(model_render_flags)/sizeof(flag_def_list);
 
-#define MAX_SUBMODEL_COLLISION_ROT_ANGLE (PI / 6.0f)	// max 30 degrees per frame
+#define MAX_SUBMODEL_COLLISION_ANGLE	(PI / 6.0f)		// max 30 degrees per frame
```

Comment:
Why did you rename this as well, since it was specific to rotation, not any generic movement?
Tbh, I'd expect this to be called something like ``MAX_SUBMODEL_COLLISION_ROT_VELOCITY``, as it's specific about rotational velocities

Reply:
I didn't like the `ROT` abbreviation, but I'm open to other terms.  I like your suggestion, so I'll use that with a small change.